### PR TITLE
Delete receiverwrapper couchdb

### PR DESCRIPTION
**Pull request by millerdev**
As promised in [the change log](https://commcare-cloud.readthedocs.io/en/latest/changelog/0081-delete-receiverwrapper-couch-db.html), the repeat records (forwarding data) is being deleted from Couch.

### Safety story

This is a cleanup step to remove unused data. Any environment that wishes to preserve this data should have already copied it to another location since they [were warned](https://commcare-cloud.readthedocs.io/en/latest/changelog/0081-delete-receiverwrapper-couch-db.html) that it would be automatically deleted at least 3 months after that announcement was made. The data has already been deleted on production.

### Automated test coverage

No.

### Migrations

- [x] The migrations in this code can be safely applied first independently of the code

### Rollback instructions

This PR has typical rollback consideration for code with migrations.

### CODE REVIEW

File: corehq/motech/repeaters/migrations/0015_drop_receiverwrapper_couchdb.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    try:
+        db.server.delete_db(db.dbname)
+    except ResourceNotFound:
+        pass
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('repeaters', '0011_remove_obsolete_entities'),
```
Comment (orangejenny) [inline corehq/motech/repeaters/migrations/0015_drop_receiverwrapper_couchdb.py:22]: Is it intentional that this doesn't depend on the most recent repeaters migration, `0014_alter_repeater_request_method`?

Comment (millerdev) [reply corehq/motech/repeaters/migrations/0015_drop_receiverwrapper_couchdb.py:22]: No. That was an oversight. Fixing.

Comment (millerdev) [reply corehq/motech/repeaters/migrations/0015_drop_receiverwrapper_couchdb.py]: https://github.com/dimagi/commcare-hq/pull/35278/commits/1968d57843b23e16e4fcd7ab7b76b9e893cad03a

**Review by orangejenny — APPROVED** [review]

**Review by orangejenny — APPROVED** [review]

File: corehq/motech/repeaters/migrations/0015_drop_receiverwrapper_couchdb.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+from django.conf import settings
+from django.db import migrations
+
+from corehq.util.django_migrations import skip_on_fresh_install
+from couchdbkit.client import Database, ResourceNotFound
+
+
+@skip_on_fresh_install
+def _delete_receiverwrapper_couchdb():
```
Comment (kaapstorm) [inline corehq/motech/repeaters/migrations/0015_drop_receiverwrapper_couchdb.py:11]: I think this needs to accept two params, `apps, schema_editor`. e.g.
https://github.com/dimagi/commcare-hq/blob/686971041bcfa6663945aaa8659d39fc860498b7/corehq/project_limits/migrations/0002_ratelimitedtwofactorlog.py#L7

**Comment by kaapstorm** [comment]
https://github.com/dimagi/commcare-hq/pull/35283

**Pull request merged**